runner/BUILD.bazel

```diff
@@ -23,20 +25,21 @@ gazelle(
     mode = "diff",
 )
 
-gazelle_binary(
+aspect_gazelle(
```

Comment:
These should always be equivalent to the `gazelle()` targets above, but with all these aspect-gazelle enhancements added such as support for `aspect run --watch //:aspect_gazelle`